cloud/azure/aks.md

```diff
@@ -1,11 +1,69 @@
 ---
 layout: page
-title: "Dynamic Provisioning on Azure Container Service (AKS)"
+title: "Azure Managed Kubernetes Service (AKS) "
 keywords: portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk, gke, gce, aks, Azure
 sidebar: home_sidebar
 ---
 
 * TOC
 {:toc}
 
-## Coming soon
+### Overview
+The [Azure Managed Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes) (aks-engine) generates ARM (Azure Resource Manager) templates for Kubernetes enabled clusters in Microsoft Azure.
+
+### Install `azure CLI`
+Install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
+
+### Login to Azure and Set Subscription
+
+* az login
+* az account set --subscription "Your-Azure-Subscription-UUID"
+
+### Create Azure Resource Group and Location
+
+Pick a name for the Azure Resource Group and choose a LOCATION value
+among the following:  
+`centralus,eastasia,southeastasia,eastus,eastus2,westus,westus2,northcentralus`
```

Comment:
is it possible to reference their doc page for a list of available regions? There is scope of this getting outdated.